Encountify/ViewModels/LocationDetailViewModel.cs

```diff
@@ -68,40 +69,44 @@ public string Category
             set => SetProperty(ref category, value);
         }
 
+        public string Address
+        {
+            get => address;
+            set => SetProperty(ref address, value);
+        }
+
         public Map Map { get; set; }
 
         public async void LoadLocationId(int Id)
         {
+            Geocoder geoCoder = new Geocoder();
             try
             {
                 location = await DataStore.GetAsync(Id);
                 Id = location.Id;
                 Name = location.Name;
                 Description = location.Description;
-                Longitude = location.Longitude;
                 Latitude = location.Latitude;
+                Longitude = location.Longitude;
+                Position position = new Position(location.Latitude, location.Longitude);
```

Comment:
Might as well already just use Latitude and Longitude rather than location.Latitude and location.Longitude